contracts/IAOLinearVesting.sol

```diff
@@ -274,7 +296,7 @@ contract IAOLinearVesting is ReentrancyGuard, Initializable {
             uint256 offeringTokenTotalHarvest, 
             uint256 offeringTokenInitialHarvest,
             uint256 offeringTokenVestedHarvest, 
-            uint256 offeringTokensVested
+            uint256 offeringTokensVesting
```

Comment:
This renaming may affect frontend integrations.

Reply:
Yeah, so this PR's coming https://github.com/ApeSwapFinance/apeswap-frontend/pull/779. it's test ready

Reply:
Nice!

Once step ahead @apewukong